### DISCUSSION TITLE
Starting CSS Club NYC

### DISCUSSION THREAD

**Discussion by tylergaw** [cssclubnyc/cssclub.nyc#3] [Announcements]
Hello friends, I'm organizing a new monthly CSS-focused meetup in New York City. I've been missing regular in-person tech meetups and I've long wanted one focused on CSS. For years, I went to as many [BrooklynJS](http://brooklynjs.com/) as I could and spoke at it twice. That's what I'm modeling CSS Club on. That event–and the surrounding BoroJS meetups–created an amazing sense of community and belonging to so many people in the NY area and beyond. I don't want to try to recreate BrooklynJS, but I want to continue on that path by creating a space for people to come together and share in something that we're passionate about.

Here's a few things on my mind as I'm getting started, please comment here or reach out with all and any ideas.

## When does this start?
This is TBD and will depend largely on finding a venue (more on that below). Once we have a venue secured, we'll hold the first one as soon as possible.

## Format
We're gonna stick with what's worked well for so many other meetups. Short talks from the community.

- Monthly on a regular day (Thinking 3rd Tuesday of the month)
- 3-4 ~15 min talks
- ~5 min between each talk. We'll end up using this space for different things, but a first idea is to give anyone from the crowd a chance to quickly talk about something
- Time for sponsors to present
- Plenty of time after talks for meeting and chatting with each other

## Talks
We want talks from everyone. Never spoken about tech in front of anyone but want to? This is 100% for you. We want to use this space to raise up new voices and to give people a chance to try out ideas in public. CSS will be our main focus, but talks can be on any technology related to styling of the web, iOS/Android apps, hardware display (no idea if styling those is a thing).

We'll likely use the BrooklynJS model of using Github issues to submit talk proposals.

## Venue
This is one of the most important things we need to get right. We're going to follow BrooklynJS's lead on this one. We're looking for:

- Somewhere in Manhattan
- Somewhere that feels unmistakably like NYC (I have my eye on a couple spaces already)
- Size, this will be based on interest, but we will keep it small on purpose, likely to max at 50 people
- Not at the office of a tech company. For many of the [same reasons as BrooklynJS](https://github.com/jed/building-brooklynjs#venue)

## Sponsors

We will need financial support to make this happen. I'll be doing outreach. If you're interested in sponsoring please reach out. contact@cssclub.nyc

## Help
I can't and don't want to do this by myself. I'll quickly bring in co-organizers, co-hosts, and monthly volunteers. This is important first, because I'll get worn out doing everything. Second, I want this to be a community event. It's important that the face of CSS Club isn't just me. Let me know if you're interested in helping out and we'll talk about what capacity. I'll also be reaching out directly to people that have done this before.

**Comment by jed** [comment]
this is so exciting, can't wait to crash your first event!

**Reply by tylergaw** [reply]
Yeah! Will be excited to see you there!

**Comment by achou11** [comment]
@tylergaw thanks for taking the initiative on this! Excited for something like this to exist, as I've admittedly been falling behind on all of the cool CSS happening nowadays.

Not sure what my capacity is but given that I'm based in Manhattan, may have some interest in helping to organize to a certain extent. Will say that hesitantly for now but in case it's useful :)

**Reply by tylergaw** [reply]
Thanks Andrew! I hear you on capacity/hesitancy haha, time is hard to come by. We'll need help now and in the future at many different capacities, so don't feel like you have to jump in now. Thinking we may get together a call for people interested in helping just to talk through what's needed. Will keep you updated.

**Comment by brendastorer** [comment]
Hi @tylergaw! I've also been contemplating the same idea for years, but never had the initiative to get anything started. I've helped with a few NYC meetups in the past and what you've written resonates with me and the type of community I'd like to be a part of again. I'm based in Manhattan and am interested in getting involved if you are looking for help. Feel free to reach out, I'm @brendamarienyc on Twitter.

**Reply by tylergaw** [reply]
Hey Brenda! I hear ya, I've been thinking on this for a while too. I was also a semi-frequent guest at ManhattanJS and went to what I think was the last CSS Layout Club. I'd love to work with you on this. I'll shoot you a DM on Twitter

**Comment by ddamato** [comment]
I'm very excited for this, I never missed a CodePen meetup when we were having them regularly.

**Reply by tylergaw** [reply]
Excellent, looking forward to seeing you there!

**Comment by rafegoldberg** [comment]
Would be psyched to help out!

**Reply by tylergaw** [reply]
Yes! Awesome, mentioned above, but we'll likely get an initial call(s) or similar together for everyone interested in helping. Will keep you updated.

**Comment by dleatherman** [comment]
Very excited about this one and definitely ready to help out!

**Reply by tylergaw** [reply]
Thanks Dan!

**Comment by chrisgalll** [comment]
really good idea, very sad that I am so far from this place(

**Comment by tylergaw** [comment]
Following up here. We have our first event scheduled for August 15th. Details on the site.

At the risk of having everyone join their 10,000th Discord...we have one now https://discord.gg/EsctDWSTwM Hop in and say hello, especially if you're interested in helping out.

**Reply by ddamato** [reply]
Tried accepting the invite to the discord but I am unable.

**Reply by tylergaw** [reply]
I think I might have goofed a bit on the Discord invite URL. The ones I was generating expire, had to set it up as a community server. Still getting used to Discord. Try this one https://discord.gg/EsctDWSTwM

That one should never expire. Updating the site now.
